.github/workflows/ai-pr-reviewer.yml

```diff
@@ -20,7 +20,7 @@ jobs:
   review:
     runs-on: ubuntu-latest
     steps:
-      - uses: Vagif12/ai-pr-reviewer@final
+      - uses: Vagif12/ai-pr-reviewer@new
```

Comment:
<img src="https://avatars.githubusercontent.com/u/116506391?s=400&u=09ae00df628c1f06039a246d04cccb2b4042ed0b&v=4" alt="Image description" width="20" height="20">   CodeSweep

The change from `Vagif12/ai-pr-reviewer@final` to `Vagif12/ai-pr-reviewer@new` indicates that a new version of the action is being used. However, it's recommended to use semantic versioning for GitHub actions to better track the versions and changes. Consider changing `@new` to a version number like `@v1.0.1`.
```diff
- uses: Vagif12/ai-pr-reviewer@new
+ uses: Vagif12/ai-pr-reviewer@v1.0.1
```


<!-- This is an auto-generated comment by OSS CodeSweep -->